chart/values.yaml

```diff
@@ -435,6 +435,22 @@ cleanStatsCache:
   expiredTimeIntervalSeconds: 600
   subfolderPattern: "*"
 
+deleteObsoleteCache:
+  enabled: true
+  log:
+    level: "info"
+  action: "delete-obsolete-cache"
+  error_codes_to_retry: ""
+  schedule: "0 */3 * * *"
+  # every 3 hours
+  nodeSelector: {}
+  resources:
```

Comment:
you don't set the resources in prod either. Will the job have enough resources? (I'm not sure if it's important, I think we don't need much anyway)

Reply:
I think it will be enough as the default (Like in cleanHfDatasetsCache, cleanDuckdbIndexCache, cleanDuckdbIndexDownloads, cleanDuckdbIndexJobRunner, cleanStatsCache, etc).
Maybe at the very first time, it would have needed more resources but since it will run every 3 hours I think it will me removing at most about 100 datasets which is not critical. 
I fixed the nodeSelector in the prod.yaml (I forgot it).